spec/mutations/message_create_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Invitation Create", type: :request do
+  include AuthHelper
+  include JsonHelper
+
+  def query
+    %(
+      mutation MessageCreate($message: String!) {
+        messageCreate(input: {
+          message: $message
+        }) {
+          message {
+            id
+          }
+          errors
+        }
+      }
+    )
+  end
+
+  let(:room_playlist_record) { create(:room_playlist_record) }
+  let(:room) { create(:room, current_record: room_playlist_record) }
+  let(:current_user) { create(:user, active_room: room) }
+
+  describe "success" do
+    it "creates a new message" do
+      graphql_request(query: query, variables: { message: "Heyo" }, user: current_user)
```

Comment:
And here's how we pass them in to the query.  We'll want to update the rest of the queries and mutations to use this pattern eventually.